pkg/export/export.go

```diff
@@ -70,6 +70,10 @@ var (
 		Name: "gcm_export_samples_sent_total",
 		Help: "Number of exported samples sent to GCM.",
 	})
+	samplesSentErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "gcm_export_samples_sent_errors_total",
```

Comment:
```suggestion
		Name: "gcm_export_samples_send_errors_total",
```

First, this metric should not be compared with gcm_export_samples_sent_total, since one is samples, and the other is requests.  "send errors" is also a more reasonable unit.